doc-content/enterprise-only/cases/case-management-designing-IT-hardware-proc.adoc

```diff
@@ -37,21 +37,31 @@ image::cases/itorders-orderhardware-process.png[IT Orders case design]
 
 +
 . Click *Ok* to open the process designer.
-. Define process-level and case-level variables.
-.. Click *BUSINESS PROCESSES (LEGACY)* -> *orderhardware*.
-.. Click image:cases/left-arrows.png[] on the right side of the designer to open the *Properties (BPMN-Diagram)* panel.
-.. Click the down arrow in the *Variable Definitions* field to open the *Editor for Variable Definitions*.
-.. Click *Add Variable*, and input the following values and click *Ok*:
+. Define the case file variables values that are accessible to the subprocesses, subcases, and business rules used in the case.
```

Comment:
Define values for the case file variables that are...

Reply:
Corrected